.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,30 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          days-before-issue-stale: 14
+          days-before-issue-close: 7
+          stale-issue-label: 'stale'
+          exempt-issue-labels: 'not-stale'
+          stale-issue-message: 'This issue is stale because it has been open 14 days with no activity. Remove stale label or comment or this will be closed in 7 days.'
```

Comment:
What will happen for current issues opened? Will this add a comment for 300+ of them? I don't think we want that :see_no_evil:. Can we avoid this for old issues?

Reply:
Great question! The [`start-date`](https://github.com/actions/stale?tab=readme-ov-file#start-date) was set in this PR to help avoid this situation. The bot will only process issues after the date. As we become more comfortable with it we can move the date backwards.